eth/v1alpha1/node.proto

```diff
@@ -62,47 +47,54 @@ service Node {
             get: "/eth/v1alpha1/node/services"
         };
     }
-
-    // Retrieve the list of peers currently connected to this node.
-    rpc ListPeers(google.protobuf.Empty) returns (Peers) {
-        option (google.api.http) = {
-            get: "/eth/v1alpha1/node/peers"
-        };
-    }
-}
-
-// Information about the current network sync status of the node.
-message SyncStatus {
-    // Whether or not the node is currently syncing.
-    bool syncing = 1;
 }
 
-// Information about the genesis of Ethereum 2.0.
-message Genesis {
-    // UTC time specified in the chain start event in the deposit contract.
-    google.protobuf.Timestamp genesis_time = 1;
-
-    // Address of the deposit contract in the Ethereum 1 chain.
-    bytes deposit_contract_address = 2;
+message NodeInfo {
```

Comment:
These info counterpoint to what we already offer today:
https://github.com/prysmaticlabs/ethereumapis/blob/master/eth/v1alpha1/beacon_chain.proto#L393

Any reason to have them re-defined in a new message? I'd like to avoid disparity if possible

Reply:
Yes there's some overlap here.  I was going to suggest that this supercede ChainHead but then we also have StreamChainHead and this PR doesn't provide streaming functionality.

There's a balance between having lots of endpoints doing similar/same thing, and having to make lots of calls to obtain basic info.  The options I see are:

  - keep the two calls; some duplication, but each has its uses and keeps existing features whilst retaining the one-shot node info call
  - remove ChainHead; may confuse people looking for the existing X / StreamX style calls
  - remove this data from NodeInfo; results in people having to make multiple calls for basic information

This are in my *vague* order of preference, but honestly I'm okay with any of these and none of them are a perfect solution.  Happy to take a steer on this one.

Reply:
pinging @rauljordan @prestonvanloon @nisdas to gather one last input before moving on